test/outputs/run_test.log.in

```diff
@@ -27,7 +27,7 @@ Exception in thread "main" java.io.FileNotFoundException: /root/.bashrc (Permiss
 	at Test.main(Test.java:513) [file:@CMAKE_BINARY_DIR@/test/Test.class]
 executable: @CMAKE_BINARY_DIR@/test/Test.class
 Caught exception java.net.UnknownHostException in method java.net.Inet4AddressImpl.lookupAllHostAddr()
-Exception in thread "main" java.net.UnknownHostException: xyzzy: Name or service not known
+Exception in thread "main" java.net.UnknownHostException: xyzzy: Temporary failure in name resolution
```

Comment:
I'm not so sure about this. I'm still getting "Name or service not known" even in Fedora 32 with `java-11-openjdk` donwgraded all the way so I don't think the message just got changed recently. Rather, I get the impression that `java.net.UnknownHostException` can be thrown with various error messages in various situations and somehow we're getting now this and now that.  
  
Of course, this being Java, I can't seem to find any documentation on what can throw the exception, under what conditions and with what messages.

Reply:
So, as @msrb has noted, some build environments don't allow networking at all which might be why we might be getting different reasons for the UnknownHostException in various situations. This might not be directly related but still struck me as interesting:    
  
`.github/workflows/check.yml: _JAVA_OPTIONS: -Djava.net.preferIPv6Addresses=true`  
  
vs.  
  
`test/CMakeLists.txt: JAVA_OPTIONS -Djava.net.preferIPv4Stack=true`